usaspending_api/disaster/v2/views/agency/spending.py

```diff
@@ -146,7 +147,9 @@ def total_queryset(self):
                     Case(
                         When(
                             self.final_period_submission_query_filters,
-                            then=F("gross_outlay_amount_by_program_object_class_cpe"),
+                            then=F("gross_outlay_amount_by_program_object_class_cpe")
```

Comment:
In the ticket `gross_outlay_amount_by_tas_cpe` is listed as the first field in the calculation under **Django ORM** section. `GrossOutlayAmountByProgramObjectClass_CPE` is the name under the **DAIMS** section, which appears to match with the field used here. 

This appears to be correct because the DAIMS formula came straight from the word doc, but I wanted to point out the discrepancy just in case.

Reply:
will update ticket